Makefile

```diff
@@ -68,6 +68,13 @@ OVNDBS_CR           ?= ${OPERATOR_BASE_DIR}/ovn-operator/${OVNDBS}
 OVNNORTHD           ?= config/samples/ovn_v1beta1_ovnnorthd.yaml
 OVNNORTHD_CR        ?= ${OPERATOR_BASE_DIR}/ovn-operator/${OVNNORTHD}
 
+# Ovs
+OVS_IMG             ?= quay.io/openstack-k8s-operators/ovs-operator-index:latest
+OVS_REPO            ?= https://github.com/openstack-k8s-operators/ovs-operator.git
+OVS_BRANCH          ?= main
+OVS                 ?= config/samples/ovs_v1beta1_ovs.yaml
```

Comment:
just to NOTE, the default do not work as it is due to additional interface requirements https://github.com/openstack-k8s-operators/ovs-operator/blob/main/config/samples/ovs_v1beta1_ovs.yaml#L12-L14

But that can be taken care seperately. PR looks good.